admiral/pkg/apis/admiral/v1/type.go

```diff
@@ -55,3 +55,29 @@ type GlobalTrafficPolicyList struct {
 
 	Items []GlobalTrafficPolicy `json:"items"`
 }
+
+//generic cdr object to wrap the GlobalTrafficPolicy api
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type RoutingPolicy struct {
+	meta_v1.TypeMeta   `json:",inline"`
+	meta_v1.ObjectMeta `json:"metadata"`
+	Spec               model.RoutingPolicy `json:"spec"`
```

Comment:
https://github.com/istio-ecosystem/admiral/tree/master/admiral/pkg/apis/admiral/model does not have routingpolicy 
Do I miss anything ?

Reply:
It is added as a part of this PR.